CoreAnalyticView/src/au/gov/asd/tac/constellation/views/analyticview/analytics/ClusterAnalyticPlugin.java

```diff
@@ -40,6 +41,7 @@
  *
  * @author cygnus_x-1
  */
+@PluginInfo(tags = {"ANALYTIC"})
 public abstract class ClusterAnalyticPlugin extends AnalyticPlugin<ClusterResult> {
```

Comment:
I probably wouldn't bother tagging an abstract class since realistically we're only over going to call implementations of this class (this is more a type of plugin as appose to a plugin itself)